archive-map.json

```diff
@@ -261,8 +262,20 @@
     {
       "appId": "mobile-toolbox",
       "iOS": 0,
-      "android": 0,
-      "allOf": [
+      "android" : 0,
+      "default" : {
```

Comment:
> Added a "default" property for the default mapping to use when the assessment is not explicitly called out in the list of assessments. The reasoning is that we do not know what the assessment identifier will be for surveys so we can't use it to map them.

Why not add this under `.["apps"]["anyOf"]`? Do you anticipate adding additional file types to `.["apps"]["default"]`?

Reply:
Because "anyOf" refers to "files that this app may add to *any* archive, including (but not limited to) those listing in the "assessments" section" whereas "default" refers to "files that should be included in an assessment that is *not* included in the "assessments" section."

in other words, we tack on the background recorders to the MTB assessments *but* an MTB assessment is not expected to include "assessmentResult.json", that is only for our assessments (ie. surveys). This is basically to work around allowing researchers to add surveys that they create. Those surveys are built and configured on bridge. This hardcoded mapping doesn't have any way to know what assessment identifiers are added to a study to support surveys and cannot map an assessmentIdentifier to an expected file manifest.

Reply:
By the by, naming things is hard and I'm open to suggestions for another name than "default". I couldn't think of a better name. Sry.

Reply:
That makes sense. "default" is fine. I can't think of a better place to put it.